python/packages/autogen-ext/pyproject.toml

```diff
@@ -42,7 +49,7 @@ dev-dependencies = []
 [tool.ruff]
 extend = "../../pyproject.toml"
 include = ["src/**", "tests/*.py"]
-exclude = ["src/autogen_ext/agents/web_surfer/*.js"]
+exclude = ["src/autogen_ext/agents/web_surfer/*.js", "src/autogen_ext/agents/video_surfer/_action_space.py"]
```

Comment:
🤔

Reply:
How do I stop it from complaining about type errors in a file that contains mostly external libs?